metadata-service/restli-client/src/main/java/com/linkedin/entity/client/JavaEntityClient.java

```diff
@@ -285,4 +304,69 @@ public SearchResult filter(@Nonnull String entity, @Nonnull Filter filter, @Null
         int start, int count, @Nonnull String actor) throws RemoteInvocationException {
         return _entitySearchService.filter(entity, filter, sortCriterion, start, count);
     }
+
+    @SneakyThrows
+    @Override
+    public VersionedAspect getAspect(@Nonnull String urn, @Nonnull String aspect, @Nonnull Long version,
```

Comment:
It'd make sense to call this one "getVersionedAspect"

Reply:
Also it's interesting to note that the RestliEntityClient version of this method will throw a Restli exception if the aspect isn't present, but this implementation will not..

Reply:
See answer above ^ I think its ok to not throw an error here since this method title does not indicate that an error should be thrown on no aspect